install.sh

```diff
@@ -231,21 +231,46 @@ do_install_rpm() {
                 ;;
         esac
     fi
-    cat <<-EOF >"/etc/yum.repos.d/rancher-rke2-${1}.repo"
-[rancher-rke2-common-${1}]
+    case "${INSTALL_RKE2_CHANNEL}" in
+        v*.*)
+            # We are operating with a version-based channel, so we should parse our version out
+            rke2_majmin=$(echo "${INSTALL_RKE2_CHANNEL}" | sed -E -e "s/^v([0-9]+\.[0-9]+).*/\1/")
+            rke2_rpm_channel=$(echo "${INSTALL_RKE2_CHANNEL}" | sed -E -e "s/^v[0-9]+\.[0-9]+-(.*)/\1/")
+            ;;
+        *)
+            rke2_majmin="1.18"
```

Comment:
Hmm, we're working on releasing 1.19, why are we hardcoding 1.18? Should this be a channel server lookup or something?

Reply:
This is just keeping behavior to what we have today. I think when we release `v1.19` we can look at dynamically choosing the versions.